mscore/shortcut.cpp

```diff
@@ -789,6 +789,17 @@ Shortcut Shortcut::_sc[] = {
          Qt::WindowShortcut,
          ShortcutFlags::A_CMD
          },
+      {
+         MsWidget::SCORE_TAB,
+         STATE_NORMAL | STATE_NOTE_ENTRY,
+         "pitch-up-diatonic-alterations",
+         QT_TRANSLATE_NOOP("action","Diatonic Up (Keep degree alterations"),
```

Comment:
IIRC we need title case here, so ` QT_TRANSLATE_NOOP("action","Diatonic Up (Keep Degree Alterations"),

Reply:
Is that for both of them or just the first?

Reply:
It looks like the shortcut text doesn't need title case (based on looking at the shortcut list), so I'll do that unless you correct me. I.e., the next line "Diatonic pitch up (Keep degree alterations)" seems to be okay.

Reply:
The 1st of each

Reply:
Ok, just re-strickened everything based on the conversations. Thanks. 
P.S. Do you know what the first QT_TRANSLATE_NOOP entry is for? I didn't necessarily understand its purpose but guessed it had to do with xml file information.

Reply:
It eneables the translations of those strings, but without doing it at that spot, as it won't work in an intialization. This way the real translations happens at its use

Reply:
Thanks